src/connection/server_support.c

```diff
@@ -2635,6 +2635,15 @@ css_process_server_server_connect (SOCKET master_fd)
       ps_Gl.set_active_tran_server_connection (std::move (chn));
       ps_Gl.start ();
       break;
+    case cubcomm::server_server::CONNECT_PASSIVE_TRAN_TO_PAGE_SERVER:
+      // *INDENT-OFF*
+      if (!chn.send_int (static_cast<int> (cubcomm::server_server::CONNECT_PASSIVE_TRAN_TO_PAGE_SERVER)))
+        {
+          er_set (ER_ERROR_SEVERITY, ARG_FILE_LINE, ER_NET_TRANSACTION_SERVER_CONNECTION, 0);
```

Comment:
Do we need to log different messages for ATS and PTS? The messages would be logged on different machines anyway.

Reply:
I don't think this is a requirement for this patch. May be changed later.